app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -975,14 +944,6 @@ private void setTextColorbyPercentage(TextView text, int value) {
         }
```

Comment:
> Maybe the best way to deal with this case in particular is to extract it into a helper class?

> EDIT: I just realized that all the color code stuff is related to the IV Results IVCombinations class, so maybe we can drop it there.

Since you sent a PR with copy-pasted code (knowingly), I'm afraid I'll have to repeat the following after millions have said it.

We should NOT copy-paste code, not even temporarily, unless there's a big reason, a big warning in both places to keep them in sync, and probably even an issue to fix it. And Copy-paste = bugs and all that. Yes we already have this problem, let's not make that worse.

I think one copy is just never used and can be dropped. If that didn't work (in general), this method can be reused as is because it can be made static (1e71958567220c5a8fdd0b6bdeb77c95bc6c99f7). If even that failed, an helper class could be used (and maybe is a good idea anyway, but can be deferred).